src/SamplesApp/UITests.Shared/Windows_Storage/Pickers/FolderPickerTests.xaml.cs

```diff
@@ -12,7 +12,7 @@
 namespace UITests.Windows_Storage.Pickers
 {
 	[Sample("Windows.Storage", ViewModelType = typeof(FolderPickerTestsViewModel), IsManualTest = true,
-		Description = "Allows testing all features of FolderPicker. Currently not supported on Android, iOS, macOS and GTK. Not selecting a folder should not cause an exception")]
+		Description = "Allows testing all features of FolderPicker. Currently not supported on Android, iOS, and macOS. Not selecting a folder should not cause an exception")]
```

Comment:
Actually the description is very out of date :-D . The only unsupported target is now WPF